scripts/python/install-pep-514.reg

```diff
@@ -0,0 +1,17 @@
+Windows Registry Editor Version 5.00
+
+[HKEY_CURRENT_USER\Software\Python\PythonCore]
+"DisplayName"="Official Python installed with Scoop"
```

Comment:
Where is this displayed?

Reply:
I'm following [PEP-514]( https://www.python.org/dev/peps/pep-0514/) and including a minimal set of the fields. In theory it's displayed by tools that look up python location in the registry. In practice, I don't have tools that display this name.